docs/src/guides/setup-dev.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+main() {
+   say "Setting up the development environment for ZKsync"
+
+   # All necessary stuff
+   say "Installing apt dependencies..."
+   sudo apt update
+   sudo apt install --yes git build-essential pkg-config cmake clang lldb lld libssl-dev libpq-dev apt-transport-https ca-certificates curl software-properties-common
+
+   # Rust
+   say "Installing Rust..."
+   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
+   source "$HOME/.cargo/env"
+   # For running unit tests
+   say "Installing Nextest..."
+   cargo install cargo-nextest --locked
+   # SQL tools
+   say "Installing SQLx CLI..."
+   cargo install sqlx-cli --version 0.8.1
+
+   # Install Docker
+   say "Installing Docker..."
+   curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+   sudo add-apt-repository --yes "deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable"
+   sudo apt install --yes docker-ce
+
+   # Start Docker
+   say "Starting Docker..."
+   sudo systemctl start docker
```

Comment:
AFAIU the usermod change is just to be able to run docker commands without sudo. In fact, in my test machine, docker is running after this command.

Reply:
Probably user on your test machine has root access, and ability to run docker commands without sudo is super important, so it must be a part of initialization.

Reply:
Why is it important?

Reply:
Because it's a major security prerequisite. Giving docker root access is a bad idea (and in general, you should never give sudo access to anything unless there is a strong request for it). And, well, it's just not convenient to have always run commands with sudo. Plus it will break e.g. `zkstack` CLI because it will (correctly) run docker commands without sudo.

Reply:
Regarding security, I'd argue exactly the reverse. With sudo you have to input a password for any docker command, thus requiring human approval. If you add the user to the docker group you effectively allow any app with non-root privileges access to the docker commands.
https://www.reddit.com/r/docker/comments/syngw7/to_sudo_or_not_to_sudo_that_is_the_question/